cilium/cmd/bpf_endpoint_delete.go

```diff
@@ -18,6 +18,10 @@ var bpfEndpointDeleteCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
```

Comment:
just curious if you have tried the below :thinking: 

```
	Args:  cobra.ExactArgs(1),
	Run: func(cmd *cobra.Command, args []string) {
```

Reply:
I didn't! I should have remembered cobra had it.